// fonts.js/lib/fonts/lines.js
// Generated by CoffeeScript 1.9.3
(function() {
  var i, len, ref, s;

  module.exports = {
    lineCount: 6,
    newLineChar: 'n',
    notSupportedChar: " ______ n|      |n|      |n|      |n|      |n|______|",
    A: "     __     n    /  \\    n   / /\\ \\   n  / /__\\ \\  n / ______ \\ n/_/      \\_\\",
    B: " _____  n|  __ \\ n| |__| |n|  __  |n| |__| |n|_____/ ",
    C: "  _____  n / ___ \\ n| /   \\_|n| |    _ n| \\___/ |n \\_____/ ",
    D: " _____   n|  __ \\  n| |  \\ \\ n| |   | |n| |__/ / n|_____/  ",
    E: " ______ n|  ____|n| |__   n|  __|  n| |____ n|______|",
    F: " ______ n|  ____|n| |__   n|  __|  n| |     n|_|     ",
    G: " ______ n|  ____|n| |  __ n| | |_ |n| |__| |n|______|",
    H: " _    _ n| |  | |n| |__| |n|  __  |n| |  | |n|_|  |_|",
    I: " _______ n|__   __|n   | |   n   | |   n __| |__ n|_______|",
    J: " _______ n|__   __|n   | |   n   | |   n __| |   n|____|   ",
    K: " _   __n| | / /n| |/ / n|    | n| |\\ \\ n|_| \\_\\",
    L: " _     n| |    n| |    n| |    n| |___ n|_____|",
    M: " _________ n|  _   _  |n| | | | | |n| | | | | |n| | | | | |n|_| |_| |_|",
    N: " ______ n|  __  |n| |  | |n| |  | |n| |  | |n|_|  |_|",
    O: " _______ n|  ___  |n| |   | |n| |   | |n| |___| |n|_______|",
    P: " _____ n|  _  |n| |_| |n|  ___|n| |    n|_|    ",
    Q: " _______ n|  ___  |n| |   | |n| | |\\| |n| |_|  _|n|____\\_\\ ",
    R: " _____ n|  _  |n| |_| |n|    _|n| |\\ \\ n|_| \\_\\",
    S: "  ______ n /  ___ \\n | |___\\/n \\___  \\ n/\\___| | n\\_____/  ",
    T: " _______ n|__   __|n   | |   n   | |   n   | |   n   |_|   ",
    U: " _     _ n| |   | |n| |   | |n| |   | |n| |___| |n|_______|",
    V: "__        __n\\ \\      / /n \\ \\    / / n  \\ \\  / /  n   \\ \\/ /   n    \\__/    ",
    W: " _   _   _ n| | | | | |n| | | | | |n| | | | | |n| |_| |_| |n|_________|",
    X: "__    __n\\ \\  / /n \\ \\/ / n  |  |  n / /\\ \\ n/_/  \\_\\",
    Y: "__    __n\\ \\  / /n \\ \\/ / n  \\  /  n  / /   n /_/    ",
    Z: "______ n\\  ___|n \\ \\   n  \\ \\  n __\\ \\ n|_____\\",
    " ": "         n         n         n         n         n         ",
    ".": "   n   n   n   n _ n|_|",
    "!": " _ n| |n| |n|_|n _ n|_|",
    "?": "  ________ n / ______ \\n/_/  ___/ /n    /____/ n    _      n   |_|     ",
    ":": "   n _ n|_|n   n _ n|_|",
    ",": "   n   n   n   n __n/_/",
    ";": "   n _ n|_|n   n __n/_/",
    "'": "  __n / /n/_/ n    n    n    ",
    '"': "  __  __n / / / /n/_/ /_/ n        n        n        ",
    '1': '  ___    n /   |   n/_/| |   n   | |   n __| |__ n|_______|',
    '2': ' _______ n|_____  |n _____| |n|  _____|n| |_____ n|_______|',
    '3': ' _______ n|_____  |n _____| |n|_____  |n _____| |n|_______|',
    '4': ' _    _ n| |  | |n| |__| |n|____  |n     | |n     |_|',
    '5': ' ______ n|  ____|n| |____ n|____  |n ____| |n|______|',
    '6': ' _____ n|  ___|n| |___ n|  _  |n| |_| |n|_____|',
    '7': ' _____ n|___  |n   / / n  / /  n / /   n/_/    ',
    '8': ' _____ n|  _  |n| |_| |n|  _  |n| |_| |n|_____|',
    '9': ' _____ n|  _  |n| |_| |n|___  |n    | |n    |_|',
    '0': ' _____ n|  _  |n| | | |n| | | |n| |_| |n|_____|',
    '/': '     __n    / /n   / / n  / /  n / /   n/_/    ',
    '<': '    ____n __/ __/n/  _/   n| |_    n\\__ \\__ n   \\___\\',
    '>': '____    n\\__ \\__ n   \\_  \\n    _| |n __/ __/n/___/   ',
    '[': ' ___ n|  _|n| |  n| |  n| |_ n|___|',
    ']': ' ___ n|_  |n  | |n  | |n _| |n|___|',
    '\\': '__     n\\ \\    n \\ \\   n  \\ \\  n   \\ \\ n    \\_\\',
    '{': '  ____n /  _/n | |  n< <   n | |_ n \\___\\',
    '}': '____  n\\_  \\ n  | | n   > >n _| | n/___/ ',
    '|': ' _ n| |n|_|n _ n| |n|_|',
    '`': '__ n\\_\\n   n   n   n   ',
    '~': '  ____    __n / __ \\__/ /n/_/  \\____/ n            n            n            ',
    '@': ' _______ n|  ___  |n| |  _| |n| | |___|n| |______n|_______/',
    '#': '   _   _   n _| |_| |_ n|_   _   _|n _| |_| |_ n|_   _   _|n  |_| |_|  ',
    '$': ' ___|____ n/  _|___ \\n| |_|___\\/n\\___|_  \\ n/\\__|_| | n\\___|___/ ',
    '%': ' _   __n|_| / /n   / / n  / /  n / / _ n/_/ |_|',
    '^': '  /\\  n /  \\ n/_/\\_\\n      n      n      ',
    '&': ' _____   n/  _  \\  n| |_| |__n > _ </ /n| |_|  < n\\____/\\_\\',
    '*': '  __  __  n _\\ \\/ /_ n|__    __|n  /_/\\_\\  n          n          ',
    '(': '  __n / /n/ / n| | n\\ \\ n \\_\\',
    ')': '__  n\\ \\ n \\ \\n | |n / /n/_/ ',
    '_': '          n          n          n          n ________ n|________|',
    '+': '       n   _   n _| |_ n|_   _|n  |_|  n       ',
    '-': '         n         n _______ n|_______|n         n         ',
    '=': '         n _______ n|_______|n _______ n|_______|n         '
  };

  ref = 'qwertyuiopasdfghjklzxcvbnm'.split('');
  for (i = 0, len = ref.length; i < len; i++) {
    s = ref[i];
    module.exports[s] = module.exports[s.toUpperCase()];
  }

}).call(this);
